composer.json

```diff
@@ -16,23 +16,22 @@
     ],
     "require": {
         "composer/installers": "^1.9",
-        "drupal/admin_toolbar": "^3.2",
+        "drupal/admin_toolbar": "^3.3",
         "drupal/autotitle": "^1.2",
-        "drupal/core-composer-scaffold": "^9.4",
-        "drupal/core-project-message": "^9.4",
-        "drupal/core-recommended": "^9.4",
+        "drupal/core-composer-scaffold": "9.5.3",
```

Comment:
This should be "^9" instead for all core. A Drupal minor core update should only touch the composer.lock file https://www.drupal.org/docs/updating-drupal/updating-drupal-core-via-composer

Reply:
done